apps/laboratory/src/components/Ethers/EthersTests.tsx

```diff
@@ -6,8 +7,17 @@ import { EthersTransactionTest } from './EthersTransactionTest'
 import { EthersWriteContractTest } from './EthersWriteContractTest'
 
 export function EthersTests() {
+  const [ready, setReady] = React.useState(false)
   const { isConnected } = useWeb3ModalAccount()
 
+  React.useEffect(() => {
```

Comment:
let's avoid usingn this, is bad practice for SSR. If we have hydration errors we should fix it in the SDK